src/coreComponents/physicsSolvers/simplePDE/LaplaceBase.hpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+#ifndef GEOSX_PHYSICSSOLVERS_SIMPLEPDE_LAPLACE_BASE_HPP
+#define GEOSX_PHYSICSSOLVERS_SIMPLEPDE_LAPLACE_BASE_HPP
+
+#include "codingUtilities/EnumStrings.hpp"   // facilities for enum-string conversion (for reading enum values from XML input)
+#include "physicsSolvers/SolverBase.hpp"  // an abstraction class shared by all physics solvers
+
+namespace geosx
+{
+
+// Like most physics solvers, the Laplace solver derives from a generic SolverBase class.
+// The base class is densely Doxygen-commented and worth a look if you have not done so already.
+// Most important system assembly steps, linear and non-linear resolutions, and time-stepping mechanisms
+// are implemented at the SolverBase class level and can thus be used in Laplace without needing reimplementation.
+
+class LaplaceBase : public SolverBase
+{
+public:
+  // The default nullary constructor is disabled to avoid compiler auto-generation:
+  LaplaceBase() = delete;
+
+  // The constructor needs a user-defined "name" and a parent Group (to place this instance in the tree structure of classes)
+  LaplaceBase( const string & name,
+               Group * const parent );
+
+  // Destructor
```

Comment:
These are all not deoxygen comments. use ///

Reply:
I turned them to Doxygen comments